package.json

```diff
@@ -59,8 +59,7 @@
     "react-router-dom": "5.3.0",
     "redux": "4.1.2",
     "regenerator-runtime": "0.13.9",
-    "tinymce": "^5.10.2",
-    "tinymce-language-selector": "^1.0.4"
```

Comment:
Curious -- Any particular reason we dropped `tincymce-language-selector`? Didn't need it after all?

Reply:
it was in another PR -- this was the wrong version of of the selector tool. the correct package is above it.